apiserver/apiserver.go

```diff
@@ -0,0 +1,96 @@
+package apiserver
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"istio.io/manager/model"
+
+	restful "github.com/emicklei/go-restful"
+	"github.com/golang/glog"
+)
+
+const (
+	Kind      = "kind"
+	Name      = "name"
+	Namespace = "namespace"
+)
+
+var (
+	schema model.ProtoSchema
```

Comment:
unused variable? we have a linter script `bin/check.sh` that can help with finding dead code.